python/tests/vo_test.py

```diff
@@ -45,6 +45,13 @@ def test_vo_example(self):
     items = vo.get_samples()
     self.assertEqual(len(items), k)
 
+    count = 0
+    for tuple in vo:
+      sample = tuple[0]
+      weight = tuple[1]
```

Comment:
worth summing weights and ensuring they equal the total?

Reply:
This seems to be a bit tricky since it is not exact. So I would suggest not test sketch correctness here.

Reply:
Discussed offline. It's exact modulo machine precision with the arithmetic, but that's a fair point. It is reasonable to test that part only in C++.